pkg/pkg/update/update.go

```diff
@@ -30,7 +30,7 @@ func NewUpdater(ghReleases GitHubReleases) Updater {
 }
 
 // Run updates the package manifest with the latest releases.
-func (u Updater) Run(pkgManifestFilename string, selectedPackagesInput []common.Package, opts Options) error {
+func (u Updater) Run(pkgManifestFilename string, selectedPackagesInput []common.Package, opts Options, workingDirectory string) error {
```

Comment:
Same as above: using workingDirectory first reads more naturally
```suggestion
func (u Updater) Run(workingDirectory string, pkgManifestFilename string, selectedPackagesInput []common.Package, opts Options) error {
```